app/src/main/java/com/hedvig/app/feature/loggedin/ui/LoggedInActivity.kt

```diff
@@ -25,15 +28,8 @@ import com.hedvig.app.feature.welcome.WelcomeDialog
 import com.hedvig.app.feature.welcome.WelcomeViewModel
 import com.hedvig.app.feature.whatsnew.WhatsNewDialog
 import com.hedvig.app.feature.whatsnew.WhatsNewViewModel
-import com.hedvig.app.util.extensions.monthlyCostDeductionIncentive
-import com.hedvig.app.util.extensions.observe
-import com.hedvig.app.util.extensions.setupLargeTitle
-import com.hedvig.app.util.extensions.showShareSheet
-import com.hedvig.app.util.extensions.startClosableChat
-import com.hedvig.app.util.extensions.view.remove
-import com.hedvig.app.util.extensions.view.setHapticClickListener
-import com.hedvig.app.util.extensions.view.show
-import com.hedvig.app.util.extensions.view.updatePadding
+import com.hedvig.app.util.extensions.*
```

Comment:
I'm guessing that your IDE does this automatically.

I'd like you to add `ktlint` to your IDE: https://github.com/pinterest/ktlint

and possibly also add some form of `Save Actions` where you can enable the IDE to perform code formatting on save. This will make sure that the codebase stays tidy over time 🙂